examples/from_googlesheets.py

```diff
@@ -0,0 +1,27 @@
+"""Example of using PandasAI with GoogleSheets"""
+
+import pandas as pd
+from data.sample_dataframe import dataframe
+
+from pandasai import PandasAI
+from pandasai.helpers.datasource import from_google_sheets
+from pandasai.llm.openai import OpenAI
+
+# instructions: https://docs.google.com/document/d/1uAmXI3e8Wm26lZQb0bYIJaXRdZi5WLJKNJHT59QMGEc/edit?usp=sharing
```

Comment:
Great job with the instructions. Would you mind adding them as part of the documentation? You can find it in the `/docs` folder!